src/db/migrations/20201031155243-change-id-name.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+module.exports = {
+  up: async queryInterface => {
```

Comment:
Why are we changing this ah? Usually, `id` is the default.

Reply:
Not sure either but its listed as a task on trello

Reply:
The column names shouldn't change. Just need to change the JSON keys that are returned when the api is called.